### PR TITLE
HCS related replacements

### PR DESCRIPTION
**Pull request by danielehc**
Adding HCS replacements for guides.

PRs affected:

* https://github.com/hashicorp/learn/pull/1396
* https://github.com/hashicorp/learn/pull/1398
* https://github.com/hashicorp/learn/pull/1407
* https://github.com/hashicorp/learn/pull/1408
* https://github.com/hashicorp/learn/pull/1431

### CODE REVIEW

**Comment by danielehc** [comment]
@kaitlincarter-hc you can review terminology for those variables before the merge. I left all suggestion un-committed  so you can easily identify them in the PRs.

File: input.yml [hunk 1]
```diff
@@ -7,3 +7,9 @@ replacements:
   '<your-consul-ip>': '10.1.2.4'
   '<your-BIG-IP-virtual-ip>': '10.1.2.5'
   '<your-authorization-header>' : 'YWxhZGRpbjpvcGVuc2VzYW1l'
+  '<External endpoint URI for your HCS datacenter>' : 'https://www.consul.io'
```
Comment (kaitlincart) [inline input.yml]:
```suggestion
  '<External endpoint URI for your HCS on Azure datacenter>' : 'https://www.consul.io'
```

File: input.yml [hunk 2]
```diff
@@ -7,3 +7,9 @@ replacements:
...
+  '<Your Azure Subscription ID>' : '06f0a6e4-ce89-b7a4-8429-4d5f987ff3c1'
```
Comment (kaitlincart) [inline input.yml]:
```suggestion
  '<Your Azure subscription ID>' : '06f0a6e4-ce89-b7a4-8429-4d5f987ff3c1'
```

File: input.yml [hunk 3]
```diff
@@ -7,3 +7,9 @@ replacements:
...
+  '<The Resource Group you used>' : 'resource_group'
```
Comment (kaitlincart) [inline input.yml]:
```suggestion
  '<The resource group>' : 'resource_group'
```

File: input.yml [hunk 4]
```diff
@@ -7,3 +7,9 @@ replacements:
...
+  '<Your Consul Managed App name>' : 'learnlab'
```
Comment (kaitlincart) [inline input.yml]:
```suggestion
  '<Your HCS on Azure instance name>' : 'learnlab'
```

File: input.yml [hunk 5]
```diff
@@ -7,3 +7,9 @@ replacements:
...
+  '<The name you used for the Consul Cluster>' : 'consul-learn-test'
```
Comment (kaitlincart) [inline input.yml]:
```suggestion
  '<The Consul datacenter name>' : 'consul-learn-test'
```

**Review by topfunky — APPROVED** [review]
I hadn't seen this before but it makes sense. Approved!

**Pull request merged**